src/main/resources/application.yaml

```diff
@@ -28,7 +28,7 @@ postgres:
   maxPoolIdleTimeMs: 1000
   maxPoolSize: 20
 
-songScoreClient:
+ariaclient:
```

Comment:
Your probably aware of this, but just noting this change will need chart update.